### PR TITLE
Add disconnect method

### LINKED ISSUE #82: Add disconnect method
We should add a method do the opposite of the `connect`-method, ie. cleanup after threads and sockets.

### PR DESCRIPTION
**Pull request by sindrehan**
**Description**

 
Fixes #82 

Remaining:
- [x] ~~Consider if we should differentiate between connecting and taking control of the drone~~
    We should do this, but I'll do that in a separate branch to get this feature out the door.
- [x] ~~Related to above, consider if we should stop the state watcher thread when disconnecting~~
    Choosing to not do this. The statewatcher thread is a daemon thread and will be cleaned up by Python's garbage collector.

#### Checklist before merging
- [x] Run the tests while connected to a drone

### CODE REVIEW

File: tests/test_sdk.py [hunk 1]
```diff
@@ -304,18 +304,11 @@ def test_wait_for_udp_com_raises_ConnectionError_on_timeout(mocker):
...
-    from blueye.protocol.exceptions import NoConnectionToDrone
-
-    mocked_pioneer._tcp_client.connect.side_effect = NoConnectionToDrone("", "")
-    with pytest.raises(ConnectionError):
-        mocked_pioneer._establish_tcp_connection()
-
-
-def test_establish_tcp_connection_ignores_RuntimeErrors(mocked_pioneer):
-    mocked_pioneer.connection_established = False
+def test_connect_ignores_repeated_starts_on_watchdog_thread(mocked_pioneer):
```
Comment (aksell) [inline tests/test_sdk.py:307]: I don't quite get how we are testing this here :thinking:, could you explain it? I can't see that we are ignoring `RunTimeErrors`s anymore in `connect`, so is the assert for that in the last three lines of this test still the behavior we want?

**Review by aksell — APPROVED** [review]
I think this looks good :+1: (I did not try out the changes on a drone)

**Comment by aksell** [comment]
Tested disconnect on a drone now and it works well for me.
I get a failing camera test when running the test whilst connected to a drone. Do you also get this @sindrehan ?
image

File: tests/test_sdk.py (hunk 1 above)
Comment (sindrehan) [reply tests/test_sdk.py:307]: We're still ignoring the RuntimeErrors, I've just separated it into its own function:
https://github.com/BluEye-Robotics/blueye.sdk/blob/16db1e1ef8ece00ea28101a92cdb912dedab10ff/blueye/sdk/pioneer.py#L200-L208

**Comment by sindrehan** [comment]
In version 1.7 of the drone software the option to set the resolution to 480p has been removed. I'll create a separate issue for updating blueye.protocol and the docstring for that function.

**Pull request merged**